examples/regression/2-advanced-analysis/plot_both_uncertainties.py

```diff
@@ -86,20 +89,52 @@ def get_1d_data_with_normal_distrib(
         ("linear", LinearRegression()),
     ]
 )
+polyn_model_quant = Pipeline(
+    [
+        ("poly", PolynomialFeatures(degree=degree_polyn)),
+        ("linear", QuantileRegressor(
+            alpha=1e-9,
+            fit_intercept=False,
+            solver="highs",
+            )),
+    ]
+)
+
 
 # Estimating prediction intervals
-Params = TypedDict("Params", {"method": str, "cv": int})
 STRATEGIES = {
-    "jackknife_plus": Params(method="plus", cv=-1),
-    "jackknife_minmax": Params(method="minmax", cv=-1),
-    "cv_plus": Params(method="plus", cv=10),
-    "cv_minmax": Params(method="minmax", cv=10),
+    "jackknife_minmax": {"method": "minmax", "cv": -1},
+    "cv_minmax": {"method": "minmax", "cv": 10},
```

Comment:
Keep `jackknife_plus` and `cv_plus` as they are the "standard" methods in the Foygel-Barber paper.

Reply:
Done, now there are 6. If you think only showing 4 is better I can get rid of of the minmax methods.

Reply:
Yes, let's remove the minmax ones.

Reply:
done